src/applications/vaos/project-cheetah/redux/actions.js

```diff
@@ -203,6 +222,96 @@ export function updateFacilitySortMethod(sortMethod, uiSchema) {
   };
 }
 
+export function getAppointmentSlots(startDate, endDate, forceFetch = false) {
+  return async (dispatch, getState) => {
+    const state = getState();
+    const useVSP = selectFeatureVSPAppointmentNew(state);
+    const siteId = getSiteIdFromFakeFHIRId(
+      selectProjectCheetahFormData(state).vaFacility,
+    );
+    const newBooking = selectProjectCheetahNewBooking(state);
+    const { data } = newBooking;
+
+    data.typeOfCareId = '301';
+    data.clinicId = 'var983_455';
+
+    const startDateMonth = moment(startDate).format('YYYY-MM');
+    const endDateMonth = moment(endDate).format('YYYY-MM');
+
+    let fetchedAppointmentSlotMonths = [];
+    let fetchedStartMonth = false;
+    let fetchedEndMonth = false;
+    let availableSlots = [];
+
+    if (!forceFetch) {
+      fetchedAppointmentSlotMonths = [
+        ...newBooking.fetchedAppointmentSlotMonths,
+      ];
+
+      fetchedStartMonth = fetchedAppointmentSlotMonths.includes(startDateMonth);
+      fetchedEndMonth = fetchedAppointmentSlotMonths.includes(endDateMonth);
+      availableSlots = newBooking.availableSlots || [];
+    }
+
+    if (!fetchedStartMonth || !fetchedEndMonth) {
+      let mappedSlots = [];
+      dispatch({ type: FORM_CALENDAR_FETCH_SLOTS });
+
+      try {
+        const startDateString = !fetchedStartMonth
+          ? startDate
+          : moment(endDate)
+              .startOf('month')
+              .format('YYYY-MM-DD');
+        const endDateString = !fetchedEndMonth
+          ? endDate
+          : moment(startDate)
+              .endOf('month')
+              .format('YYYY-MM-DD');
+
+        const fetchedSlots = await getSlots({
+          siteId,
+          typeOfCareId: data.typeOfCareId,
```

Comment:
Rather than set `typeOfCareId` in `data` above, I would just pass `TYPE_OF_CARE_ID` here.

Reply:
There is that const you were referring to!